nntrainer/graph/graph_node.h

```diff
@@ -74,7 +74,7 @@ class GraphNode {
    *
    * @return list of name of the nodes which form input connections
    */
-  virtual const std::vector<std::string> &getInputConnections() const = 0;
+  virtual const std::vector<std::string> getInputConnections() const = 0;
```

Comment:
I wonder why this change is needed?

Reply:
returning & is only valid if we have actual std::vector<std::string> now we have wrapper's around this we can't return the reference to it.